### PR TITLE
MAINTAINERS: add 2, remove 5

### PR DESCRIPTION
**Pull request by vbatts**
We have a stalled project, as we have a number of maintainers have moved
on. We thank you all for your time.

Going forward, I'm nominating 2 contributors that have a vested
interest:
* @thaJeztah
* @AkihiroSuda

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

**Comment by vbatts** [comment]
@opencontainers/go-digest-maintainers

**Comment by AkihiroSuda** [comment]
LGTM, thanks

**Comment by stevvooe** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/opencontainers/go-digest/pull-request/54/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=opencontainers/go-digest)

File: MAINTAINERS [hunk 1]
```diff
@@ -1,8 +1,5 @@
-Aaron Lehmann <aaronl@vitelus.com> (@aaronlehmann)
-Brendan Burns <bburns@microsoft.com> (@brendandburns)
 Derek McGowan <derek@mcgstyle.net> (@dmcgowan)
-Jason Bouzane <jbouzane@google.com> (@jbouzane)
-John Starks <jostarks@microsoft.com> (@jstarks)
-Jonathan Boulle <jon@nstack.com> (@jonboulle)
 Stephen Day <stephen.day@docker.com> (@stevvooe)
```
Comment (AkihiroSuda) [inline MAINTAINERS:2]: @stevvooe Your email address needs to be updated?

Comment (vbatts) [reply MAINTAINERS:2]: ooh yes
i'll look up the gmail one

**Comment by dmcgowan** [comment]
LGTM

**Comment by AkihiroSuda** [comment]
ping @opencontainers/go-digest-maintainers

**Comment by dmcgowan** [comment]
I think we should go ahead and merge this (looks like @stevvooe 's approve got unset after push though). Getting a quorum here of existing maintainers doesn't make sense considering the original purpose of this PR and there is no explicit rule requiring a majority to approve. We could have the TOB vote instead, but I don't think there is any issue here and if there is it can still be brought up to the TOB later.

**Comment by dmcgowan** [comment]
@caniszczyk can you be the judge here?

**Comment by dmcgowan** [comment]
Let's get this merged as there seems to be no issue, the TOB can resolve any issues in that unlikely scenario.

**Pull request merged**